xiv_stats.php

```diff
@@ -773,7 +815,7 @@ function getValueOrZero($data, $key) {
           <div class="col s12 m6" style="width:100%;">
               <div class="card white">
                   <div class="card-content black-text">
-                      <span class="card-title black-text light">OTHER</span>
+                      <a id="other"><span class="card-title black-text light">OTHER</span></a>
```

Comment:
Suggest renaming this to "Misc Stats" as it's been defined in the drop-down, or visa versa.